bundle/bin/rke2-killall.sh

```diff
@@ -62,7 +62,6 @@ systemctl stop rke2-agent.service || true
 killtree $({ set +x; } 2>/dev/null; getshims; set -x)
 
 do_unmount_and_remove '/run/k3s'
-do_unmount_and_remove "${RKE2_DATA_DIR}"
```

Comment:
We removed for kill all and added for uninstall since we do not want to clean the directory because it can break the cluster, for uninstall make sense since we do want to delete everything.